rtnl/addr.go

```diff
@@ -87,7 +90,7 @@ func (c *Conn) Addrs(ifc *net.Interface, family int) (out []*net.IPNet, err erro
 			iplen := len(m.Attributes.Address)
 			a := &net.IPNet{
 				IP:   m.Attributes.Address,
-				Mask: net.CIDRMask(int(m.PrefixLength), iplen),
+				Mask: net.CIDRMask(int(m.PrefixLength), iplen<<3),
```

Comment:
On mobile, I am not sure what this bitshift means here. Could you add a comment that explains it?

Reply:
You're right. Let me rephrase the code.